my_enumerable.rb

```diff
@@ -0,0 +1,26 @@
+module MyEnumerable
+  def all?
+    @list.each do |n|
+      return false unless yield n
+    end
+
+    true
+  end
+
+  def any?
+    @list.each do |n|
+      return true if yield n
+    end
+
+    false
+  end
+end
+
+def filter
+  new_list = []
+  @list.each do |n|
+    new_list.push(n) if yield n
+  end
+
+  new_list
+end
```

Comment:
- [ ] Fantastic progress on your work, While reviewing your code I noticed your `filter` method wasn't working as expected, so I conducted a thorough testing of the `filter` method. Initially, everything seemed fine, and the implementation appeared correct. However, upon closer testing, I identified the root cause of the issue.

   - It turns out that the `filter` method is currently residing outside the `MyEnumerable` module. This has led to a situation where the method is being treated as a private entity and isn't behaving as intended.

   - 🛠 To address this concern, I recommend moving the `filter` method inside the `MyEnumerable` module. This adjustment should seamlessly integrate the method with the module, resolving the private method recognition issue.

Happy coding! 😊🚀🍀